lib/query_string_search/abstract_matcher.rb

```diff
@@ -1,6 +1,6 @@
 module QueryStringSearch
   class AbstractMatcher
-    attr_accessor :attribute, :value
+    attr_accessor :attribute, :desired_value
```

Comment:
Is attribute the best name for the attribute?  It is somewhat confusing.  I don't have any suggestions so attribute may be the best option.